deployment/stacks/ui-auth.ts

```diff
@@ -0,0 +1,380 @@
+import { Construct } from "constructs";
+import {
+  aws_cognito as cognito,
+  aws_iam as iam,
+  aws_lambda as lambda,
+  aws_lambda_nodejs as lambda_nodejs,
+  aws_wafv2 as wafv2,
+  aws_ssm as ssm,
+  Aws,
+  Duration,
+  custom_resources as cr,
+  RemovalPolicy,
+} from "aws-cdk-lib";
+import { WafConstruct } from "../constructs/waf";
+import { IManagedPolicy } from "aws-cdk-lib/aws-iam";
+import { isLocalStack } from "../local/util";
+
+interface CreateUiAuthComponentsProps {
+  scope: Construct;
+  project: string;
+  stage: string;
+  isDev: boolean;
+  applicationEndpointUrl: string;
+  restApiId: string;
+  customResourceRole: iam.Role;
+  iamPath: string;
+  iamPermissionsBoundary: IManagedPolicy;
+  oktaMetadataUrl: string;
+  oktaOidcClientId?: string;
+  oktaOidcClientSecret?: string;
+  oktaOidcIssuer?: string;
+  bootstrapUsersPassword?: string;
+  secureCloudfrontDomainName?: string;
+  userPoolDomainPrefix?: string;
+  sesSourceEmailAddress?: string;
+  attachmentsBucketArn: string;
+}
+
+export function createUiAuthComponents(props: CreateUiAuthComponentsProps) {
+  const {
+    scope,
+    project,
+    stage,
+    isDev,
+    applicationEndpointUrl,
+    restApiId,
+    customResourceRole,
+    iamPath,
+    iamPermissionsBoundary,
+    oktaMetadataUrl,
+    oktaOidcClientId,
+    oktaOidcClientSecret,
+    oktaOidcIssuer,
+    bootstrapUsersPassword,
+    secureCloudfrontDomainName,
+    userPoolDomainPrefix,
+    // TODO: note that sesSourceEmailAddress - CreateEmailConfiguration in SLS was being configured based on SSM parameters sesSourceEmailAddress which was not set in any environment.
```

Comment:
Let's rip it out. This was true in HCBS and I ripped out. People won't miss it since it never did anything.